CONTRIBUTING.md

```diff
@@ -15,6 +15,7 @@ Apollo is built on top of several opensource tecnologies:
 * [Consul](http://consul.io) for service discovery, DNS
 * [Docker](http://docker.io) for application container runtimes
 * [Weave](https://github.com/zettio/weave) for networking of docker containers
+* [Zookeeper](https://zookeeper.apache.org/) for cconfiguration information, naming, providing distributed synchronization, and providing group services
```

Comment:
I'll change it to * [Zookeeper](https://zookeeper.apache.org/) to maintain cluster state for Mesos in the next commit?